<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>log_display</class>
 <widget class="QMainWindow" name="log_display">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1279</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>40</y>
      <width>221</width>
      <height>501</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="11" column="2">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="1" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_6"/>
     </item>
     <item row="8" column="1" colspan="2">
      <widget class="QPushButton" name="all_search_but">
       <property name="text">
        <string>查看全部日志</string>
       </property>
      </widget>
     </item>
     <item row="14" column="2">
      <widget class="QPushButton" name="exit_but">
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout"/>
     </item>
     <item row="2" column="1" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>账号</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="user_id_edit"/>
       </item>
       <item>
        <widget class="QPushButton" name="user_id_search_but">
         <property name="text">
          <string>查找</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="1" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>书号</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="book_id_edit"/>
       </item>
       <item>
        <widget class="QPushButton" name="book_id_search_but">
         <property name="text">
          <string>查找</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="log_text">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>521</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>21</y>
      <width>791</width>
      <height>541</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/picture/框Big.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>-250</x>
      <y>360</y>
      <width>621</width>
      <height>381</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/picture/22334.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>log_text</zorder>
   <zorder>label_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1279</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
